bridge-message-predicates/contract-message-receiver/src/lib.sw

```diff
@@ -1,15 +1,6 @@
 library contract_message_receiver_abi;
 
 abi MessageReceiver {
-    /// Process a message passed to the receiver contract to facilitate a deposit of bridged tokens.
```

Comment:
No more docs?

Reply:
There are no docs in the actual predicate repo. This is supposed to be a carbon copy of what's in the predicate repo and hopefully will eventually just be included in the std lib, or be part of our own forc version of crates.io for cargo. The existing docs were overly specific to using messages for bridging so I thought I'd just get rid of it for now instead of rewording it